importAllData

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+set -u
+./museum/importObjects -r -p -m
```

Comment:
`-m` has an exit 0 in it, as a hangover from previous imports. There is a use case for loading an import mapping without actually running it (ie you want to leave it there for others to run), so this makes sense. I'll experiment with `getopts` and see whether you can run flags together.